config/configgrpc/configgrpc.go

```diff
@@ -62,7 +62,7 @@ type ClientConfig struct {
 	Compression configcompression.Type `mapstructure:"compression"`
 
 	// TLSSetting struct exposes TLS client configuration.
-	TLSSetting configtls.TLSClientSetting `mapstructure:"tls"`
+	TLSSetting configtls.ClientConfig `mapstructure:"tls"`
```

Comment:
We should at least open a PR about re-naming the variable as well.

Reply:
Sure. I can do that. 
How many versions are these "old" field names kept for?

Reply:
Until `1.0`?

Reply:
I've created an issue to track this: #9548

Reply:
Right now the old names are usually kept around for a single release. You can see the schedule here: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/release.md#release-schedule.